<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyTotalRecForm</class>
 <widget class="QWidget" name="ModifyTotalRecForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnSubmit">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>680</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>630</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>480</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>740</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>471</y>
     <width>691</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInsert">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>530</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>580</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>231</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>User Total Play Record</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
